lock-api-objects/src/main/java/com/palantir/lock/watch/LockWatchStateUpdate.java

```diff
@@ -33,6 +34,7 @@
     @JsonSubTypes.Type(value = LockWatchStateUpdate.Success.class, name = LockWatchStateUpdate.Success.TYPE),
     @JsonSubTypes.Type(value = LockWatchStateUpdate.Snapshot.class, name = LockWatchStateUpdate.Snapshot.TYPE)
 })
+@DoNotMock
```

Comment:
I've never seen this before - why do we need it?

Reply:
Discussed - fine, I buy it.

Reply:
Because mocking this makes tests explode since even the noop implementations depend on being able to read stuff